.travis.yml

```diff
@@ -0,0 +1,22 @@
+---
+language: node_js
+node_js:
+  - "0.12"
+
+sudo: false
+
+cache:
+  directories:
+    - node_modules
+
+before_install:
+  - "npm config set spin false"
+  - "npm install -g npm@^2"
+
+install:
+  - npm install -g bower
+  - npm install
+  - bower install
+
+script:
+  - npm test
```

Comment:
That came with `ember init`.  I didn't bother deleting it since it would just have to be deleted every time.  I suppose we could add it to .gitignore.  It's also good inspiration for our own circleci config.